lib/cell.rb

```diff
@@ -0,0 +1,18 @@
+class Cell
+  attr_reader :coordinate, :ship, :empty
+
+  def initialize(coordinate)
+    @coordinate = coordinate
+    @ship = nil
+    @empty = true
+  end
+
+  def empty?
+    @empty
+  end
+
+  def place_ship(ship)
```

Comment:
Nice, so the place_ship method says when its called on the individual cell will not be empty and contain a ship:)